testing/scikitlearn/test2.py

```diff
@@ -0,0 +1,73 @@
+from sklearn.model_selection import KFold
```

Comment:
Do you want to add comments outlining what to test? Would it be better to generate all of this code as you tested?

Reply:
This kinda feels like what you should end up with.